paymentsheet/src/main/java/com/stripe/android/paymentsheet/DeferredIntentValidator.kt

```diff
@@ -125,6 +139,21 @@ internal object DeferredIntentValidator {
 
         return firstFingerprint == secondFingerprint
     }
+
+    private fun validatePaymentMethodOptionsSetupFutureUsage(
```

Comment:
Should we try/catch or runCatching this?

What if something about the structure changes in the future, we'd want to fail, but not crash.